orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/support/TargetServerGroup.groovy

```diff
@@ -169,7 +169,8 @@ class TargetServerGroup {
     static Location locationFromStageData(StageData stageData) {
       try {
         List zones = stageData.availabilityZones?.values()?.flatten()?.toArray()
-        return resolveLocation(stageData.namespace, stageData.region, zones?.getAt(0))
+        String region = stageData.destination?.region?:stageData.region
```

Comment:
Isn't this used to determine where to locate the server group that we are cloning? I think "target" is misleading in this context.

Reply:
when I did a search for the usages of `locationFromStageData` it appears this method is only used here [AbstractDeployStrategyStage](https://github.com/spinnaker/orca/blob/90e24bd548777c2c0bc1f55163ec2a4f15d3f131/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/AbstractDeployStrategyStage.groovy)

on L130
https://github.com/spinnaker/orca/blob/90e24bd548777c2c0bc1f55163ec2a4f15d3f131/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/AbstractDeployStrategyStage.groovy#L130

and L230

https://github.com/spinnaker/orca/blob/90e24bd548777c2c0bc1f55163ec2a4f15d3f131/orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/strategies/AbstractDeployStrategyStage.groovy#L230

setting the region to destination to the destination region would mean we are applying the deployment strategy to the where the clone destination is.